content/docs/implementation-notes.md

```diff
@@ -838,61 +837,61 @@ function mountTree(element, containerNode) {
+* React menempatkan informasi tentang pembaruan saat ini ke objek internal yang disebut "transaksi". Transaksi berguna untuk melacak antrean *method lifecycle* yang tertunda, DOM saat ini bersarang untuk peringatan, dan hal lain yang bersifat "global" untuk pembaruan tertentu. Transaksi juga memastikan React "membersihkan semuanya" setelah pembaruan. Misalnya, kelas transaksi yang disediakan oleh React DOM mengembalikan pilihan masukan setelah pembaruan apapun.
 
-### Jumping into the Code {#jumping-into-the-code}
+### Melompat ke Kode {#jumping-into-the-code}
 
-* [`ReactMount`](https://github.com/facebook/react/blob/83381c1673d14cd16cf747e34c945291e5518a86/src/renderers/dom/client/ReactMount.js) is where the code like `mountTree()` and `unmountTree()` from this tutorial lives. It takes care of mounting and unmounting top-level components. [`ReactNativeMount`](https://github.com/facebook/react/blob/83381c1673d14cd16cf747e34c945291e5518a86/src/renderers/native/ReactNativeMount.js) is its React Native analog.
-* [`ReactDOMComponent`](https://github.com/facebook/react/blob/83381c1673d14cd16cf747e34c945291e5518a86/src/renderers/dom/shared/ReactDOMComponent.js) is the equivalent of `DOMComponent` in this tutorial. It implements the host component class for React DOM renderer. [`ReactNativeBaseComponent`](https://github.com/facebook/react/blob/83381c1673d14cd16cf747e34c945291e5518a86/src/renderers/native/ReactNativeBaseComponent.js) is its React Native analog.
-* [`ReactCompositeComponent`](https://github.com/facebook/react/blob/83381c1673d14cd16cf747e34c945291e5518a86/src/renderers/shared/stack/reconciler/ReactCompositeComponent.js) is the equivalent of `CompositeComponent` in this tutorial. It handles calling user-defined components and maintaining their state.
-* [`instantiateReactComponent`](https://github.com/facebook/react/blob/83381c1673d14cd16cf747e34c945291e5518a86/src/renderers/shared/stack/reconciler/instantiateReactComponent.js) contains the switch that picks the right internal instance class to construct for an element. It is equivalent to `instantiateComponent()` in this tutorial.
+* [`ReactMount`](https://github.com/facebook/react/blob/83381c1673d14cd16cf747e34c945291e5518a86/src/renderers/dom/client/ReactMount.js) adalah tempat kode seperti `mountTree()` dan `unmountTree()` dari tutorial hidup ini. Ini menangani pemasangan dan pelepasan komponen tingkat atas. [`ReactNativeMount`](https://github.com/facebook/react/blob/83381c1673d14cd16cf747e34c945291e5518a86/src/renderers/native/ReactNativeMount.js) adalah analog React Native-nya.
```

Comment:
```suggestion
* [`ReactMount`](https://github.com/facebook/react/blob/83381c1673d14cd16cf747e34c945291e5518a86/src/renderers/dom/client/ReactMount.js) adalah tempat kode seperti `mountTree()` dan `unmountTree()` dari tutorial ini berada. Ini menangani pemasangan dan pelepasan komponen tingkat atas. [`ReactNativeMount`](https://github.com/facebook/react/blob/83381c1673d14cd16cf747e34c945291e5518a86/src/renderers/native/ReactNativeMount.js) adalah analog React Native-nya.
```

"[...} from this tutorial lives" refers to the location of the code itself, not this "live tutorial".

Reply:
oh I see,. thanksss